webpack/scenes/ContentViews/components/ContentViewSelect/ContentViewSelectOption.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Flex, SelectOption } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  global_palette_black_600 as pfDescriptionColor,
+} from '@patternfly/react-tokens';
+import { uniq } from '../../../../utils/helpers';
+import ContentViewIcon from '../../../../scenes/ContentViews/components/ContentViewIcon';
+
+const relevantVersionObjFromCv = (cv, env) => { // returns the entire version object
```

Comment:
Pretty much this entire file is reused from `ChangeHostCVModal.js`. Instead of duplicating the code, I'd like to see the components and helper methods extracted and used in both places. There are a few ways to do this - you can keep everything in the ChangeHostCVModal area and then import them here; you can put all the common stuff in a new area and import them both places; or you can move everything over here and import stuff in ChangeHostCVModal. Or you could do a combination of these - whatever works best. The important thing is that we don't have two copies of the same code in our code base, so this way if one gets updated we don't have to remember to update the other.